src/components/Tree/DataZTree/components/ZTree/index.vue

```diff
@@ -376,6 +376,7 @@ export default {
   .ztree {
     width: 100%;
     overflow: auto;
+    z-index: 1000;
     height: 648px;
     background-color: #ffffff;
 
```

Comment:
There doesn't seem to be any specific code differences between the versions you've specified or the current date (2025). It's possible that "current version" is referring only to the latest code changes on this platform since I'm based on an earlier version of your request. In summary:

* No issues detected

If you need assistance with JavaScript-specific coding, please specify the problem and include relevant context/parameters where suitable.